src/test/java/uk/gov/ons/ssdc/caseprocessor/service/DeactivateUacProcessorTest.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.ons.ssdc.caseprocessor.service;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static uk.gov.ons.ssdc.caseprocessor.utils.Constants.DEACTIVATE_UAC_ROUTING_KEY;
+
+import java.util.List;
+import java.util.Map;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.test.util.ReflectionTestUtils;
+import uk.gov.ons.ssdc.caseprocessor.model.dto.DeactivateUacDTO;
+import uk.gov.ons.ssdc.caseprocessor.model.dto.ResponseManagementEvent;
+import uk.gov.ons.ssdc.caseprocessor.model.entity.Case;
+import uk.gov.ons.ssdc.caseprocessor.model.entity.UacQidLink;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DeactivateUacProcessorTest {
+
+  @Mock private RabbitTemplate rabbitTemplate;
+
+  @InjectMocks private DeactivateUacProcessor underTest;
+
+  @Test
+  public void testProcessDeactivateUacRow() {
+    // Given
+    ReflectionTestUtils.setField(underTest, "outboundExchange", "testExchange");
+
+    Case caze = new Case();
+    caze.setSample(Map.of("foo", "bar"));
+    caze.setCaseRef(123L);
```

Comment:
Test passes without these values sample/caseRef being set.  I think I'm as guilty as anyone regarding this, but we shouldn't add to it.

Reply:
we'll take them out